sql/src/test/java/io/crate/protocols/postgres/types/PGArrayTest.java

```diff
@@ -145,8 +145,12 @@ public void testDecodeUTF8Text() {
         assertThat(o, is(new Object[] {10, 20}));
 
         // ensure unquoted integer values are decoded correctly (a bug prevented that once)
-        o = pgArray.decodeUTF8Text("{10,20}".getBytes(StandardCharsets.UTF_8));
-        assertThat(o, is(new Object[] {10, 20}));
+        o = pgArray.decodeUTF8Text("{10,2}".getBytes(StandardCharsets.UTF_8));
+        assertThat(o, is(new Object[] {10, 2}));
+
+        // ensure single char element value is decoded correctly (a bug prevented that once)
```

Comment:
Is this about the single digit number 1? Maybe you can rephrase that to clarify - I was first confused if it should be 1 element only in the array.

Reply:
right, was writing this while already was away with my mind :) will change